sql/core/src/test/scala/org/apache/spark/sql/CollationSQLExpressionsSuite.scala

```diff
@@ -1584,6 +1584,234 @@ class CollationSQLExpressionsSuite
     })
   }
 
+  test("CurrentTimeZone expression with collation") {
+    // Supported collations
+    Seq("UTF8_BINARY", "UTF8_BINARY_LCASE", "UNICODE", "UNICODE_CI").foreach(collationName => {
```

Comment:
since we're now using this `Seq("UTF8_BINARY", "UTF8_BINARY_LCASE", "UNICODE", "UNICODE_CI")` a lot, let's separate it out and call it something like `testCollationsSeq`